resources/scripts/ttx_diff.py

```diff
@@ -371,6 +371,22 @@ def stat_like_fontmake(ttx):
     ver.attrib["value"] = "0x00010001"
 
 
+# https://github.com/googlefonts/fontc/issues/1107
+def normalize_glyf_contours(ttx):
+    for glyph in ttx.xpath("//glyf/TTGlyph"):
+        contours = glyph.xpath("./contour")
+        if len(contours) < 2:
+            continue
+        normalized = sorted(contours, key=lambda c: etree.tostring(c))
```

Comment:
do we even need to `tostring`? not sure how expensive it is, but all we need is some total order, can we get that from `c` itself?

Reply:
It wasn't entirely clear to me what that would do whereas tostring is simple to reason about